best-practices/tagging/index.html

```diff
@@ -303,15 +333,14 @@ <h4>Headings</h4>
 				</aside>
 				<aside class="example" title="Multiline heading">
 					<p>Note that while <code>&lt;br/></code> can be used to force multiple lines, doing so will hardcode those breaks. It is better to identify the heading with either a tag or class and use CSS to center the braille &lpar;<code>text-wrap: balance</code>&rpar;. See the <a href=https://daisy.github.io/ebraille/best-practices/styling/#line-breaking>Styling Best Practices section on line breaks</a> for more information.</p>
-					<pre><code class="html">&lt;h1>⠠⠉⠓⠁⠏⠞⠑⠗ ⠼⠛&lt;br/>
-⠠⠞⠓⠑ ⠠⠗⠑⠍⠁⠗⠅⠁⠃⠇⠑ ⠠⠞⠗⠥⠞⠓&lt;br/>
-⠁⠃⠕⠥⠞ ⠠⠚⠕⠓⠝ ⠠⠎⠍⠊⠞⠓&lt;/h1></code></pre>
+					<pre><code class="html">&lt;h1>⠠⠉⠓⠁⠏⠞⠑⠗ ⠼⠛
+&lt;p class="subtitle">⠠⠞⠓⠑ ⠠⠗⠑⠍⠁⠗⠅⠁⠃⠇⠑ ⠠⠞⠗⠥⠞⠓ ⠁⠃⠕⠥⠞ ⠠⠚⠕⠓⠝ ⠠⠎⠍⠊⠞⠓&lt;p>&lt;/h1></code></pre>
```

Comment:
Is the `<p>` supposed to be inside the `<h1>` or outside?

Reply:
It's intentionally inside. Thinking about it now, I'm going to change it to a span. There was a comment a while back that I think got me confused here. A span makes more sense, I think.

Reply:
Yes, `p` inside `h1` is not even allowed I think.

`span` might not be the most intuitive choice to represent something that gets rendered as a block (gets a dedicated line). But it is structurally correct.

Another option is to use `hgroup`. From [Accessible Publishing Knowledge Base](http://kb.daisy.org/publishing/docs/html/headings.html):

```
<hgroup>
  <h1>...</h1>
  <p role="doc-subtitle" class="subtitle">...</p>
</hgroup>
```
.
 I also suggested before:

```
<header>
  <h1>...</h1>
  <p class="subtitle">...</p>
</header>
```

It depends on the situation whether this is appropriate or not. Note that a `hgroup` could also appear inside a `header`.

Even `<br/>` can be appropriate, when the heading is broken at a specific place purely for presentational reasons, and not because it consists of two parts semantically.

Reply:
> Even `<br/>` can be appropriate, when the heading is broken at a specific place purely for presentational reasons,  > and not because it consists of two parts semantically.

I addressed this part in https://github.com/daisy/ebraille/pull/288/commits/1ec23833f0d6e413bab8ba890cfb34a413fbf591